test/e2e/apps/deployment.go

```diff
@@ -396,9 +396,6 @@ var _ = SIGDescribe("Deployment", func() {
 
 		ginkgo.By("patching the DeploymentStatus")
 		deploymentStatusPatch, err := json.Marshal(map[string]interface{}{
-			"metadata": map[string]interface{}{
-				"labels": map[string]string{"test-deployment": "patched-status"},
-			},
```

Comment:
We don't need to remove this label.

Reply:
This change is pointless, the patch operation below is ONLY applied to status subresource, so anything outside status is just misleading. That's why I removed it.